ef_tests/tests/cancun.rs

```diff
@@ -2,13 +2,9 @@ use std::path::Path;
 
 mod common;
 
-fn eip4788_tests(path: &Path) -> datatest_stable::Result<()> {
-    common::parse_and_execute_test_file(path);
+fn cancun_tests(path: &Path) -> datatest_stable::Result<()> {
+    common::parse_test_file(path);
     Ok(())
 }
 
-datatest_stable::harness!(
```

Comment:
we should `parse_and_execute_test_file` the working ones and just `parse_test_file` the rest. We can achieve this with `harness!`

Reply:
Done!